src/shared/components/Card/styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+import { StyledCardProps } from './types';
+import { theme } from '../../theme';
+import { css } from '@emotion/react';
+
+export const StyledCard = styled.section<StyledCardProps>`
+  padding: 72px 32px;
+
+  background: ${theme.colors.background.light};
+  box-shadow: ${theme.shadow};
+
+  ${({ position }) =>
+    position === 'bottom'
+      ? css`
+          border-radius: 40px 40px 0px 0px;
+        `
+      : css`
+          border-radius: 0px 0px 40px 40px;
+        `}
```

Comment:
**Suggestion:** How about this? Does it work with emotion or we always need the `css`?
```suggestion
  border-radius: ${({ position }) => position === 'bottom' 
    ? '40px 40px 0px 0px' 
    : '0px 0px 40px 40px;'}
```